pkgs/development/tools/build-managers/bazel/bazel-1/default.nix

```diff
@@ -0,0 +1,551 @@
+{ stdenv, callPackage, lib, fetchurl, fetchFromGitHub
+, runCommand, runCommandCC, makeWrapper, recurseIntoAttrs
+# this package (through the fixpoint glass)
+, bazel
```

Comment:
This is likely the wrong bazel. Not sure why we need that in here but you could change that to `bazel_1` in this file.

Reply:
Fixed, I was confused about this too, I kept that consistent with before